Marlin/src/lcd/extui/lib/mks_ui/draw_change_speed.cpp

```diff
@@ -311,11 +310,11 @@ void disp_speed_type() {
   switch (speedType) {
     case 1:
       lv_obj_set_event_cb_mks(buttonExt, event_handler, ID_C_EXT, "bmp_extruct_sel.bin", 0);
-      lv_obj_set_event_cb_mks(buttonMov, event_handler, ID_C_MOVE, "bmp_mov_changespeed.bin", 0);
+      lv_obj_set_event_cb_mks(buttonMov, event_handler, ID_C_MOVE, "bmp_mov_changeSpeed.bin", 0);
       break;
 
     default:
-      lv_obj_set_event_cb_mks(buttonExt, event_handler, ID_C_EXT, "bmp_Extruct.bin", 0);
+      lv_obj_set_event_cb_mks(buttonExt, event_handler, ID_C_EXT, "bmp_speed_extruct.bin", 0);
```

Comment:
extrude + construct = extruct?

Reply:
Right now we are just matching with their icons file name. Right now we need to keep this compatibility. They have a self hosted full version that use those icon names. And MKS haven't finished all their migration to lvgl and to marlin.